SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -71,7 +71,7 @@ class SPATIALGDK_API UEventTracingSamplingSettings : public UObject
 	GENERATED_BODY()
 public:
 	UPROPERTY(EditAnywhere, Category = "Event Tracing", meta = (ClampMin = 0.0f, ClampMax = 1.0f))
-	float SamplingProbability = 1.0f;
+	double SamplingProbability = 1.0f;
```

Comment:
Double due to the fact that the Worker event setting that is set by the variable is also a double